packages/playground/src/utils/types.ts

```diff
@@ -21,4 +21,6 @@ export interface ApplicationCard {
   icon: string;
   route: string;
   tags?: string[];
+  hasUpdate: boolean;
```

Comment:
can't get why we need `hasUpdate` also why it's required ... if it's not a prop in object so it's false